src/jaxsim/api/contact.py

```diff
@@ -395,3 +406,96 @@ def mixed_jacobian(W_H_C: jtp.Matrix, W_J_WC: jtp.Matrix) -> jtp.Matrix:
             raise ValueError(output_vel_repr)
 
     return O_J_WC
+
+
+@jax_dataclasses.pytree_dataclass
+class ContactsState(JaxsimDataclass, abc.ABC):
```

Comment:
If possible, I'd keep this a plain class that just inherits from `abc.ABC`, having all methods marked as abstract (they can also be static).

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/178/commits/6542f8bb0cbd3d0607ba4db9a909740c8da63951